tests/integration/docusaurus/connecting_to_your_data/cloud/azure/pandas/configured_python_example.py

```diff
@@ -43,13 +43,13 @@
 datasource_config["execution_engine"]["azure_options"][
     "account_url"
 ] = "superconductivetests.blob.core.windows.net"
-datasource_config["execution_engine"]["azure_options"]["credential"] = AZURE_CREDENTIAL
+datasource_config["execution_engine"]["azure_options"]["credential"] = credential
 datasource_config["data_connectors"]["configured_data_connector_name"]["azure_options"][
     "account_url"
 ] = "superconductivetests.blob.core.windows.net"
 datasource_config["data_connectors"]["configured_data_connector_name"]["azure_options"][
     "credential"
-] = AZURE_CREDENTIAL
+] = credential
```

Comment:
I don't feel particularly strongly about this but why the change from `AZURE_CREDENTIAL` to `credential`? Happy to stick with whatever you think works better here but would love to hear the reasoning for my own edification.

Reply:
@cdkini Stylistically, I did not want constants to be in this form.  If you think that this is not proper, I can revert uniformly.  Please let me know what your preference is.  Thank you.

Reply:
what is your preferred alternative for constants? Just asking since it is a convention I've used for a while (but may not necessarily be idiomatic Python).

Let's do what you think is best here but I'd love to get your thoughts just for better understanding 🐍

Reply:
Reverted to Constant.